cluster_test.go

```diff
@@ -28,6 +28,17 @@ func TestGetStatistics(*testing.T) {
 	}
 }
 
+func TestGetFloatStatistics(*testing.T) {
+	floatStatsKeyArray := []string{"cluster.disk.bytes.in.rate", "ifs.bytes.total", "cluster.disk.xfers.in.rate"}
+	stats, err := client.GetFloatStatistics(defaultCtx, floatStatsKeyArray)
+	if err != nil || len(stats.StatsList) != 2 {
```

Comment:
3 stats keys are passed to `GetFloatStatistics`. Do we expect 2 stats in list?

Reply:
Fixed